R/cwTools.R

```diff
@@ -22,7 +22,13 @@ MSW.cwt <- function (ms, scales = 1, wavelet = "mexh")
         stop("Unsupported wavelet!")
     }
     oldLen <- length(ms)
-    ms <- MSW.extendNBase(ms, nLevel = NULL, base = 2)
```

Comment:
instead of removing the old with your new code I would suggest to have here something like a:
```r
if (extendLengthMSW) {
    your new code
} else
old code
```